management/src/test/java/org/ehcache/management/providers/statistics/HitCountTest.java

```diff
@@ -0,0 +1,150 @@
+
+  @Rule
+  public final TemporaryFolder diskPath = new TemporaryFolder();
+
+  private static final long CACHE_HIT_TOTAL = 4;
+
+  private final ResourcePools resources;
+  private final List<String> statNames;
+  private final List<Long> tierExpectedValues;
+  private final Long cacheExpectedValue;
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return asList(new Object[][] {
+    //1 tier
+    { newResourcePoolsBuilder().heap(1, MB), Arrays.asList("OnHeap:HitCount"), Arrays.asList(CACHE_HIT_TOTAL), CACHE_HIT_TOTAL },
+    { newResourcePoolsBuilder().offheap(1, MB), Arrays.asList("OffHeap:HitCount"), Arrays.asList(CACHE_HIT_TOTAL), CACHE_HIT_TOTAL },
+    { newResourcePoolsBuilder().disk(1, MB), Arrays.asList("Disk:HitCount"), Arrays.asList(CACHE_HIT_TOTAL), CACHE_HIT_TOTAL },
+
+    //2 tiers
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB), Arrays.asList("OnHeap:HitCount","OffHeap:HitCount"), Arrays.asList(2L,2L), CACHE_HIT_TOTAL},
+    { newResourcePoolsBuilder().heap(1, MB).disk(2, MB), Arrays.asList("OnHeap:HitCount","Disk:HitCount"), Arrays.asList(2L,2L), CACHE_HIT_TOTAL},
+    //offheap and disk configuration below is not valid.  Throws IllegalStateException no Store.Provider found to handle configured resource types [offheap,disk]
+
+    //3 tiers
+    { newResourcePoolsBuilder().heap(1, MB).offheap(2, MB).disk(3, MB), Arrays.asList("OnHeap:HitCount","OffHeap:HitCount","Disk:HitCount"), Arrays.asList(2L,0L,2L), CACHE_HIT_TOTAL},
+    { newResourcePoolsBuilder().heap(1, ENTRIES).offheap(2, MB).disk(3, MB), Arrays.asList("OnHeap:HitCount","OffHeap:HitCount","Disk:HitCount"), Arrays.asList(1L,1L,2L), CACHE_HIT_TOTAL},
+    });
+  }
+
+  public HitCountTest(Builder<? extends ResourcePools> resources, List<String> statNames, List<Long> tierExpectedValues, Long cacheExpectedValue) {
+    this.resources = resources.build();
+    this.statNames = statNames;
+    this.tierExpectedValues = tierExpectedValues;
+    this.cacheExpectedValue = cacheExpectedValue;
+  }
+
+  @Test
+  public void test() throws InterruptedException, IOException {
+    DefaultManagementRegistryConfiguration registryConfiguration = new DefaultManagementRegistryConfiguration().setCacheManagerAlias("myCacheManager");
+    registryConfiguration.addConfiguration(new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES));
+    ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(registryConfiguration);
+
+    Configuration cacheConfiguration = new DefaultConfiguration(EvictionTest.class.getClassLoader(),
+            new DefaultPersistenceConfiguration(diskPath.newFolder()));
+
+    Collection<Service> services = new ArrayList<Service>();
+    services.add(managementRegistry);
+
+    CacheManager cacheManager = null;
+
+    try {
+      cacheManager = new EhcacheManager(cacheConfiguration, services);
```

Comment:
You are right, that would make more sense since the management tests should use the api.  I will change it.